src/osyris/plot/parser.py

```diff
@@ -33,7 +35,8 @@ def parse_layer(layer,
             vmax = params["vmax"]
             del params["vmax"]
 
-        params["norm"] = get_norm(norm=params["norm"], vmin=vmin, vmax=vmax)
```

Comment:
I would just add the check on `isinstance(str)` inside the `get_norm` function, so you only have to add it once?